frontend/src/components/organisms/TableFilters/index.jsx

```diff
@@ -53,6 +88,8 @@ const TableFilter = (props) => {
     return filter.name;
   };
 
+  const category = getFilterID(categoryFilterValue);
```

Comment:
Do we really want `category` to be "no-filters" or ""? Your logic below doesn't render the multi-select box if `category` is `false`, which seems to correspond to the old logic ... your new `category` won't ever be "false"...

Reply:
That looks *lots* better! Thanks. 🥳